packages/math/typesetter.lua

```diff
@@ -129,6 +129,8 @@ end
 local function handleMath (_, mbox, mode)
   if mode == 'display' then
     mbox.mode = b.mathMode.display
+  elseif mode == 'numbered' then
```

Comment:
instead of a new mode ("numbered"), I wonder if it wouldn't be more general to have options on the "display" mode.
E.g.
```
\math[mode=display]{...}% unnumbered
\math[mode=display, numbered=true]{...}% numbered with counter 'equation'
\math[mode=display, numbered=true, counter=xxx]{...}% numbered with (user-specified) counter id 'xxx'
\math[mode=display, numbered=true, number=N]{...}% numbered with (user-specified) value N'
```

Or would it be an overkill?

Reply:
Using options is more general, and the examples given are appealing. In fact, I had some trouble in naming a new mode, since in it the equations are the display mode, with numbers.
However, I also like the idea of not having to type a lot of options to get numbered equations. LaTeX also do something like this, with the `displaymath` and `equation` environments.
We can have several options, and provide a script, e.g. `\equation{...}`, that calls `\math[mode=display, numbered=true, counter=equation]{...}`. What do you think?

Reply:
> What do you think?

As long as we the have the core / low-level features, personally I don't have a strong opinion regarding "shortcut" convenience commands.

In my recent projects I use the SIL language less and less, favoring Markdown or Djot whenever I can (using [markdown.sile](https://github.com/Omikhleia/markdown.sile))

In Markdown (resp. Djot) there's just `$...$` and `$$...$$` (resp. the Djot corresponding syntax) for inline and display math, but one _could_ use Pandoc spans (resp. Djot direct attributes). E.g. in the Djot support, I could add something such as:

```
$$`formula`{ numbered=true }
```

Or even a pseudo-class:
```
$$`formula`{ .equation }
```

This could perhaps be interesting, hence my previous suggestion for more flexibility, but to be honest I don't type much maths usually so I am not sure what people might expect :)